service-python/featuretoggle/src/lib/main.py

```diff
@@ -0,0 +1,26 @@
+import logging
+from typing import Dict
+
+from . import utils
+
+
+def report_feature_toggles(event: Dict):
+    """
+    Take a request and return a response that includes feature toggle data
```

Comment:
A couple observations about high-level style:
- `event` isn't used in this function, which makes sense because it just returns the entire features list every time. Are you planning to add more functionality later? Regardless, I think it's fine to remove `event` for now and change the signature later if needed -- this function isn't an API that needs to maintain compatibility.
- Both this file and `utils.py` are both pretty small, and both pretty generically-named. It might be nice to just combine them into one file and give it a slightly more informative name, like `feature_toggles.py`. (Again: are there big plans on the horizon?)

Reply:
Removed event for now.
I can combine them... they were split because I was basing off of the assessclaim microservices for consistency. BIgger plans on the horizon are environment-specific feature flags, as well as hooking to a UI service for easier toggling on/off, possibly per environment as well?  Was told for this initial pass a simple boolean would be okay.